src/ray/util/process.cc

```diff
@@ -191,6 +191,13 @@ class ProcessFD {
         _exit(pid2 == -1 ? errno : 0);  // Parent of grandchild; must exit
       }
       // This is the spawned process. Any intermediate parent is now dead.
+
+      // Clear any CPU affinity that might have been inherited from the raylet.
+      // See https://github.com/ray-project/ray/issues/35984.
+      cpu_set_t mask;
```

Comment:
Is it possible to add a test for this one?

Reply:
I don't know of a good way to do it (in the PR description: This PR is not tested because the best way to do so would require sudo permissions.). What do you think?

Reply:
Oops. Sorry I missed that comment. I think it is okay to merge if you manually have checked it then.

Reply:
Lmk if you tested manually!